app/services/periodicSync.ts

```diff
@@ -118,23 +118,23 @@ export const getAllPeriodicSyncData = (): any => {
     console.log("Don't sync");
   }
   else {
-    if(childList.length>0){
-    childList.map((child: any) => {
-      const childAgedays = (DateTime.now()).diff((DateTime.fromISO(child.birthDate)), 'days').toObject().days;
-      if (childAgedays >= child.taxonomyData.days_to - child.taxonomyData.buffers_days) {
-        const i = childAge.findIndex((_item: any) => _item.id === child.taxonomyData.id);
-        // i > -1 && i < childAge.length
-        if (i > -1 && i < childAge.length - 1) {
-          const nextchildAgeData = childAge[i + 1];
-          if (nextchildAgeData.age_bracket.length > 0) {
-            nextchildAgeData.age_bracket.map((ages: any) => {
-              ageBrackets.push(ages);
-            })
+    if (childList.length > 0) {
+      childList.map((child: any) => {
```

Comment:
Since you're not returning a new array, .forEach() is the better choice over .map().